### ISSUE TITLE
Preview not available

### ISSUE THREAD

**Issue by tusharnandy** [atom-community/markdown-preview-plus#491]
Something has happened to this package in my machine. Whenever I try and click Ctrl+Shift+M, I get a black screen in the preview and nothing is displayed on it.
Please help
Thank you in advance

**Comment by tusharnandy** [comment]
I am using ubuntu by the way

**Comment by jodh-intel** [comment]
Same problem on F32.

**Comment by jodh-intel** [comment]
I'm using atom version atom-1.47.0-0.1.

Might be a core problem as I see the same problem with https://atom.io/packages/markdown-preview-enhanced.

**Comment by lierdakil** [comment]
Duplicate of https://github.com/atom-community/markdown-preview-plus/issues/487
Upstream issue: https://github.com/atom/atom/issues/20604

TL;DR: downgrade to 1.46 and wait for the next release. If you want to complain, complain to upstream. Sorry.
